build.rs

```diff
@@ -547,10 +548,18 @@ fn obj_path(out_dir: &Path, src: &Path) -> PathBuf {
 
 fn configure_cc(c: &mut cc::Build, target: &Target, c_root_dir: &Path, include_dir: &Path) {
     let compiler = c.get_compiler();
-    // FIXME: On Windows AArch64 we currently must use Clang to compile C code
-    let compiler = if target.os == WINDOWS && target.arch == AARCH64 && !compiler.is_like_clang() {
-        let _ = c.compiler("clang");
-        c.get_compiler()
+    // FIXME: On Windows AArch64 we currently must use LLVM (clang-cl) to compile C code
```

Comment:
I think we should say "clang or clang-cl" instead of "clang-cl" here. I think we should add a comment "If the compiler is MSVC, switch to clang-cl so that `CFLAGS`, etc., are more likely to be compatible."

Reply:
I had that at first, but considered that the weight is mostly on the file being compiled via an `LLVM` toolchain. Whether the compiler/argument "driver" is `clang` or `clang-cl` doesn't matter, but you're right that I could comment this better in the `match` arms.